### ISSUE TITLE
Wrong handling of paths without params

### ISSUE THREAD

**Issue by asulykos** [lwsjs/rewrite#11]
Rewrite rules having no parameters, automatically applied to every path (even to the non-matching ones.)

The problem is in **getTargetUrl** function of _util.js_.

The **toUrl** is automatically set to the **to** parameter, and will be returned even if the path doesn't match. For non-matching paths the intact **url** parameter should be returned instead.

```javascript
function getTargetUrl (from, to, url) {
  const { pathToRegexp } = require('path-to-regexp')
  const fromParams = []
  const re = pathToRegexp(from, fromParams)
  const fromMatches = re.exec(url)
  let toUrl = to // <--------- it should be 'url' if 'fromMatches' is undefined

  for (const [index, fromParam] of fromParams.entries()) {
    if (fromMatches && fromMatches[index + 1]) {
      fromParam.value = fromMatches[index + 1]
    }
  }

  /* replace named params */
  for (const fromParam of fromParams) {
    if (typeof fromParam.name === 'string') {
      if (fromParam.value) {
        toUrl = toUrl.replace(new RegExp(`:${fromParam.name}`, 'g'), fromParam.value)
      } else {
        toUrl = url
      }
    }
  }

  /* replace positional params */
  for (const fromParam of fromParams) {
    if (typeof fromParam.name !== 'string') {
      toUrl = url.replace(re, toUrl)
    }
  }

  return toUrl
}
```

**Comment by 75lb** [comment]
Hi, could you post an example of a rewrite rule I can use to reproduce this bug please?

**Comment by asulykos** [comment]
Let's consider the following rules:

```
rewrite: [
  {
    from: '/login/(.*)',
    to: 'dist/login/$1'
  },
  {
    from: '/login',
    to: 'dist/login/index.html'
  }
]
```

Here with the first rule, I'd like to rewrite every path starting with /login/ to my _dist_ folder path. This rule alone works, but as soon as I add the second one, everything will be redirected to **'dist/login/index.html'**. The second rule doesn't have params to be replaced. _getTargetUrl()_ will return the unmodified _'toUrl'_ which has been set to _'to'_ at the beginning of the function. (This 'to' variable holds **'dist/login/index.html'**.)

**Comment by 75lb** [comment]
Fixed and released in lws-rewrite v3.1.1 - please upgrade. 

Thanks, let me know if you find any more issues!

**Comment by asulykos** [comment]
Thank you very much, that was really quick. :)
